CMakeLists.txt

```diff
@@ -11,8 +11,18 @@ list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
 
 ################################################################################
 
-option(MDSPAN_ENABLE_TESTS "Enable tests." Off)
-option(MDSPAN_ENABLE_EXAMPLES "Build examples." Off)
+
+# Determine if MDSpan is built as a subproject (using add_subdirectory)
+# or if it is the master project.
+set(MASTER_PROJECT OFF)
+if(CMAKE_CURRENT_SOURCE_DIR STREQUAL CMAKE_SOURCE_DIR)
+  set(MASTER_PROJECT ON)
+  message(STATUS "CMake version: ${CMAKE_VERSION}")
+endif()
```

Comment:
This is only to prevent to build tests and examples if using as a subproject with FetchContents or as git submodule from an other cmake project.

Reply:
> This is only to prevent to build tests and examples if using as a subproject with FetchContents or as git submodule from an other cmake project.

This is how it behave previously.  Turned off by default.

> And IMHO the standalone project should build the test and its examples.

I tend to prefer the current behavior.

Reply:
OK, done